### PR TITLE
TMS-17 Dropdown and TextArea wrappers added; CaseTest and NewCasePage…

### PR DESCRIPTION
**Pull request by nastyfizzle**
… added

### CODE REVIEW

File: src/test/java/pages/NewCasePage.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.openqa.selenium.By;
+import wrappers.Dropdown;
+import wrappers.TextArea;
+
+import static com.codeborne.selenide.Selectors.byXpath;
+import static com.codeborne.selenide.Selenide.$;
+
+public class NewCasePage {
+
+    public static final String SAVE_BUTTON = "#save-case";
```
Comment (dzmitryrak) [inline src/test/java/pages/NewCasePage.java]: SAVE_BUTTON_CSS

File: src/test/java/pages/NewCasePage.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public static final By SAVE_AND_CREATE_BUTTON = byXpath("//*[contains(text(), 'Save and create another')]");
```
Comment (dzmitryrak) [inline src/test/java/pages/NewCasePage.java:14]: byXpath = By.xpath

File: src/test/java/pages/NewCasePage.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    public static final By CANCEL_BUTTON = byXpath("//*[contains(text(), 'Cancel')]");
+    public static final String TITLE_FIELD = "#title";
+    public static final String TEXT_AREA = "#undefinedGroup.form-group";
+
+    public NewCasePage isOpened() {
+        $(SAVE_BUTTON).shouldBe(Condition.visible);
+        return this;
+    }
+
+    public NewCasePage fillInFields(String title, String statusOption, String suiteOption, String severityOption,
```
Comment (dzmitryrak) [inline src/test/java/pages/NewCasePage.java]: Value object

File: src/test/java/pages/ProjectsPage.java [hunk 4]
```diff
@@ -23,8 +22,13 @@ public NewProjectPage clickOnCreateNewProjectButton() {
...
     public ProjectDetailsPage openProject(String name) {
-        $$("a.defect-title").findBy(text(name)).click();
-        $$(By.xpath("//div[contains(@id, 'react-select')]")).findBy(text("option")).click();
+        //$$(PROJECT_NAME).findBy(text(name)).click(); //не работает..у меня не получилось решить эту проблему
+        $$(PROJECT_NAME).find(Condition.text(name)).click(); //этот локатор тоже не работает
+        return new ProjectDetailsPage();
+    }
+
+    public ProjectDetailsPage openProjectByUrl() {
+        Selenide.open("https://app.qase.io/project/TP");
```
Comment (dzmitryrak) [inline src/test/java/pages/ProjectsPage.java]: openProjectByUrl(String projectCode) {
 Selenide.open("project/" + projectCode)
}

File: src/test/java/pages/NewCasePage.java (hunk 1 above)
Comment (nastyfizzle) [reply src/test/java/pages/NewCasePage.java]: fixed

File: src/test/java/pages/NewCasePage.java (hunk 2 above)
Comment (nastyfizzle) [reply src/test/java/pages/NewCasePage.java:14]: thank you :)

File: src/test/java/pages/NewCasePage.java (hunk 3 above)
Comment (nastyfizzle) [reply src/test/java/pages/NewCasePage.java]: fixed

File: src/test/java/pages/ProjectsPage.java (hunk 4 above)
Comment (nastyfizzle) [reply src/test/java/pages/ProjectsPage.java]: fixed

**Pull request merged**